src/common/casualReferences.ts

```diff
@@ -34,6 +34,15 @@ export function yesterday(reference: ReferenceWithTimezone): ParsingComponents {
     return component;
 }
 
+export function theDayBeforeYesterday(reference: ReferenceWithTimezone): ParsingComponents {
```

Comment:
This function is limited to very narrow usage. To make it more flexible, I would make `theDayBefore(reference: ReferenceWithTimezone, numDay: number)`.

I would also update yesterday
```
yesterday(reference: ReferenceWithTimezone) {
 return theDayBefore(reference, 1);
} 
```

Reply:
Switched to theDayBefore and theDayAfter in https://github.com/wanasit/chrono/pull/458/commits/824f832bd610a2aaf531977cfce4a790431f7764